media/src/main/java/io/wcm/handler/media/impl/MediaHandlerImpl.java

```diff
@@ -87,6 +87,11 @@
     return new MediaBuilderImpl(mediaRef, this);
   }
 
+  @Override
+  public @NotNull MediaBuilder get(String mediaRef, Resource contextResource) {
```

Comment:
please add the same nullability annotations as used in the interface to avoid code warnings.